packages/calendar/react/src/components/action-pane/attendees.tsx

```diff
@@ -70,7 +80,10 @@ const AddUnknownAttendee: React.FC<{ text: string; clearText: () => void }> = ({
           name: text,
           surname: "",
           email: text,
-          color: "#C4C4C4",
+          // Escape hatch to not have to modify types just to get typescript
+          // to accept our unknown color. We don't want the unknown color to be
+          // selectable on the backend, thus we don't add it to the enum.
+          color: unknownColorName as "red",
```

Comment:
this is a bit sketchy but any other solution would require either

a) separating logic for "real" and unknown attendees, or
b) widening the types in a bunch of other places and handling cases there

so i think it's fine here?